### ISSUE TITLE
Data source accessibility (Act 1)

### ISSUE THREAD

**Issue by tskarhed** [grafana/grafana#35613]
With the current accessibility initiative some key pain points have been identified. The datasources in Explore have some navigation issues.

Keyboard navigation should behave nicely in the following. Also make sure that screen readers handle these properly.

## Query Editors

- [ ] Prometheus
- [ ] Loki
- [ ] Elasticsearch

## Other low hanging fruit

- [ ] Provide alt text and title to icon buttons

**Comment by torkelo** [comment]
would be nice to create a statement page for the docs https://github.com/grafana/grafana/issues/34771

**Comment by tskarhed** [comment]
Good point! @torkelo! We'll address it in the accessibility weekly today I think
